pkg/cmd/project/view/view.go

```diff
@@ -0,0 +1,208 @@
+
+	viewCmd.Flags().StringVar(&opts.userOwner, "user", "", "Login of the user owner. Use \"@me\" for the current user.")
+	viewCmd.Flags().StringVar(&opts.orgOwner, "org", "", "Login of the organization owner.")
+	viewCmd.Flags().BoolVarP(&opts.web, "web", "w", false, "Open project in the browser.")
+	viewCmd.Flags().StringVar(&opts.format, "format", "", "Output format, must be 'json'.")
+
+	// owner can be a user or an org
+	viewCmd.MarkFlagsMutuallyExclusive("user", "org")
+
+	return viewCmd
+}
+
+func runView(config viewConfig) error {
+	if config.opts.web {
+		url, err := buildURL(config)
+		if err != nil {
+			return err
+		}
+
+		if err := config.URLOpener(url); err != nil {
+			return err
+		}
+		return nil
+	}
+
+	if config.opts.format != "" && config.opts.format != "json" {
+		return fmt.Errorf("format must be 'json'")
+	}
+
+	owner, err := queries.NewOwner(config.client, config.opts.userOwner, config.opts.orgOwner)
+	if err != nil {
+		return err
+	}
+
+	project, err := queries.NewProject(config.client, owner, config.opts.number, true)
+	if err != nil {
+		return err
+	}
+
+	if config.opts.format == "json" {
+		return printJSON(config, *project)
+	}
+
+	return printResults(config, project)
+}
+
+func buildURL(config viewConfig) (string, error) {
+	var url string
+	if config.opts.userOwner == "@me" {
+		login, err := queries.ViewerLoginName(config.client)
+		if err != nil {
+			return "", err
+		}
+		url = fmt.Sprintf("https://github.com/users/%s/projects/%d", login, config.opts.number)
+	} else if config.opts.userOwner != "" {
+		url = fmt.Sprintf("https://github.com/users/%s/projects/%d", config.opts.userOwner, config.opts.number)
+	} else if config.opts.orgOwner != "" {
+		url = fmt.Sprintf("https://github.com/orgs/%s/projects/%d", config.opts.orgOwner, config.opts.number)
+	}
+
+	return url, nil
+}
+
+func printResults(config viewConfig, project *queries.Project) error {
+
+	var sb strings.Builder
+	sb.WriteString("# Title\n")
+	sb.WriteString(project.Title)
+	sb.WriteString("\n")
+
+	sb.WriteString("## Description\n")
+	if project.ShortDescription == "" {
+		sb.WriteString(" -- ")
+	} else {
+		sb.WriteString(project.ShortDescription)
+	}
+	sb.WriteString("\n")
+
+	sb.WriteString("## Visibility\n")
+	if project.Public {
+		sb.WriteString("Public")
+	} else {
+		sb.WriteString("Private")
+	}
+	sb.WriteString("\n")
+
+	sb.WriteString("## URL\n")
+	sb.WriteString(project.URL)
+	sb.WriteString("\n")
+
+	sb.WriteString("## Item count\n")
+	sb.WriteString(fmt.Sprintf("%d", project.Items.TotalCount))
+	sb.WriteString("\n")
+
+	sb.WriteString("## Readme\n")
+	if project.Readme == "" {
+		sb.WriteString(" -- ")
+	} else {
+		sb.WriteString(project.Readme)
+	}
+	sb.WriteString("\n")
+
+	sb.WriteString("## Field Name (Field Type)\n")
+	for _, f := range project.Fields.Nodes {
+		sb.WriteString(fmt.Sprintf("%s (%s)\n\n", f.Name(), f.Type()))
+	}
+
+	// TODO: respect the glamour env var if set
+	out, err := glamour.Render(sb.String(), "dark")
```

Comment:
Protip: You can use `pkg/markdown.Render` to render markdown instead of going straight to Glamour. You can read the preferred terminal theme from `IOStreams.TerminalTheme()`

Reply:
fixed in f484b822